### DISCUSSION TITLE
Old-Style Canvas Scaling as a toggleable option?

### DISCUSSION THREAD

**Discussion by SauceDLX** [obsproject/obs-studio#11612] [General]
This might be a really niche proposal, but it would be nice to have a setting to allow sources to not rescale when switching to a profile with a different resolution, as it worked before 31. The new auto rescaling can be useful but can be a hassle when it gets applied to graphics that are meant to be pixel-perfect, so it seems like a good thing to be able to enable/disable.

**Comment by Warchamp7** [comment]
You can reset the scaling via the Scene Collection menu
